src/main/java/uk/co/ramp/people/PopulationGenerator.java

```diff
@@ -100,7 +90,7 @@ int findAge(double v, Map<Integer, Double> c, Map<Integer, MinMax> populationAge
 
         for (int i = 0; i < runProperties.populationSize(); i++) {
 
-            int age = findAge(dataGenerator.nextUniform(0, 1), cumulative, properties.populationAges());
+            int age = findAge(cumulative, properties.populationAges());
```

Comment:
We should pass in `properties.populationDistribution()` here instead of `cumulative`

Reply:
Also, strange that the unit test which tests this method is not failing here.

Reply:
Will change.